dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_multiple_addresses_saved.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
We already the same fixture
Please, check the same class, it should be used in another testcases

BTW if we create fixture we MUST create rollback
Thanks

Reply:
@naydav I haven't found it. Can you take a look and give me the name. I did look at the whole project.